geowebcache/web/src/main/webapp/WEB-INF/geowebcache-security.xml

```diff
@@ -16,6 +16,7 @@
     <security:http>
         <security:intercept-url method="GET" pattern="/rest/web/**" access="permitAll"/>
         <security:intercept-url pattern="/rest/**" access="hasRole('ROLE_ADMINISTRATOR')"/>
+        <security:intercept-url pattern="/restng/**" access="hasRole('ROLE_ADMINISTRATOR')"/>
```

Comment:
Again isn't this supposed to be replacing the old REST not sitting beside it?

Changes to context mean changes to GeoServer will be required so we need a PR downstream for that.